.circleci/config.yml

```diff
@@ -596,6 +596,46 @@ jobs:
             else
               echo "No config changes, skipping"
             fi
+  update_fenix_apks:
+    machine:
+      image: ubuntu-2004:2023.10.1
+      docker_layer_caching: true
+    working_directory: ~/experimenter
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "32:8e:72:0b:9a:a1:1c:b8:7e:90:e1:53:a3:73:68:47" # for git pushes from circleci, since relies on ssh
+      - checkout
+      - gh/setup:
+          token: GH_TOKEN # for gh commands from circleci, since relies on user token, since por que no los dos?
+      - run:
+          name: Setup Git
+          command: |
+            git config --local user.name "dataops-ci-bot"
+            git config --local user.email "dataops+ci-bot@mozilla.com"
+            gh config set git_protocol https
+      - run:
+          name: Check for Application Services update
+          command: |
+            git checkout main
+            git pull origin main
+            cd experimenter/tests/integration/nimbus/android
+            export CURRENT_FENIX_BUILD_ID=$(cat fenix-build.env | grep -oP '(?<=FIREFOX_FENIX_BUILD_ID=).*')
+            ./update-fenix-apks.sh
+            export LATEST_FENIX_BUILD_ID=$(cat fenix-build.env | grep -oP '(?<=FIREFOX_FENIX_BUILD_ID=).*')
```

Comment:
nit: this doesnt' need export if it is not going to be picked up by a third party script.